packages/ui/src/views/landing-page.tsx

```diff
@@ -0,0 +1,73 @@
+import { useNavigate } from 'react-router-dom'
+
+import {
+  Button,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+  Text
+} from '@/components'
+import { SandboxLayout } from '@/views'
+
+interface TypesSpace {
```

Comment:
doesn't this already exist on some other page? what about in the breadcrumbs?

Reply:
Tried looking, couldn't find it. Will merge for now and de-dupe at a later point.